pkg/apis/deployment/v1/tls_sni_spec.go

```diff
@@ -27,6 +27,12 @@ import (
 
 // TLSSNISpec holds TLS SNI additional certificates
 type TLSSNISpec struct {
+	// Mapping is the mapping of the SNI options.
+	// +doc/type: map[string][]string
+	// +doc/example: mapping:
+	// +doc/example:   secret:
+	// +doc/example:   - domain.example.com
+	// +doc/link: Server Name Indication|https://docs.arangodb.com/stable/develop/http-api/security/
```

Comment:
I would rather link to the startup options because there is an explanation of SNI in the details
```suggestion
	// +doc/link: Server Name Indication|https://docs.arangodb.com/stable/components/arangodb-server/options/#--sslserver-name-indication
```